sdk/src/main/java/io/radar/sdk/model/RadarGeofence.kt

```diff
@@ -35,11 +35,23 @@ class RadarGeofence(
      */
     val metadata: JSONObject?,
 
+
     /**
      * The geometry of the geofence.
      */
-    val geometry: RadarGeofenceGeometry?
-) {
+    val geometry: RadarGeofenceGeometry?,
+
+    /**
+     * The radius of the geofence.
+     */
+    val geometryRadius: Double?,
```

Comment:
We already store center and radius in the `geometry` field - so it seems like we don't need these two new fields.